extension/main.js

```diff
@@ -1,6 +1,11 @@
 (function () {
 	'use strict';
 
+	function createNotificationId() {
+		var id = Math.floor(Math.random() * 9007199254740992) + 1;
```

Comment:
> On the contrary.

Agreed. Limiting ourselves to a single notification seems like the desirable behaviour.